backend/api/Areas/Project/Mapping/Dispose/ProjectMap.cs

```diff
@@ -1,4 +1,5 @@
 using Mapster;
```

Comment:
Overall the idea is to push the decision whether to map the metadata or not to the mapper, that way a controller can still call the project service without having to deal with the performance cost of serialization.

Reply:
Perfect!